test/e2e/specs/site-editor/patterns.spec.js

```diff
@@ -1,13 +1,24 @@
 /**
  * WordPress dependencies
  */
-const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+const {
+	test: base,
+	expect,
+} = require( '@wordpress/e2e-test-utils-playwright' );
+
+/** @type {ReturnType<typeof base.extend<{patterns: Patterns}>>} */
+const test = base.extend( {
+	patterns: async ( { page }, use ) => {
+		await use( new Patterns( { page } ) );
+	},
+} );
```

Comment:
@Mamaduka You might be interested to know how we can assign types and auto-complete to file-level fixtures in JS files. This is the simplest way I found. Unfortunately we were using `.use` which isn't possible AFAIK. We might want to recommend `.extend` from now on.

That said, this is totally optional though. It's for power users who like types and auto-complete in their code editors 😅 .

Reply:
Oh, this is a nice one. Bookmarking it. Thanks, @kevin940726!

Maybe we can mention this in advance or a similar section. As you said, probably not every test needs it.